File: src/main/java/com/ciazhar/authserver/dto/request/ChangeRoleForm.java
package com.ciazhar.authserver.dto.request;

/**
 * Created by ciazhar on 6/23/17.
 */

public class ChangeRoleForm {
    private String id;
    private String role;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }
}

File: src/main/kotlin/com/ciazhar/authserver/model/dto/request/ChangeRoleForm.kt
package com.ciazhar.authserver.model.dto.request

import javax.validation.constraints.NotNull

/**
 * Created by ciazhar on 6/23/17.
 */


class ChangeRoleForm (

        @field:NotNull
        var id: String?=null,

        @field:NotNull
        var role: String?=null
)
